variables.tf

```
@@ -45,6 +45,8 @@ variable "subnets" {
 
     service_endpoints = optional(list(string), [])
 
+    service_endpoint_policy_ids = optional(list(string), null)
```

Group related properties:

```suggestion
    service_endpoints           = optional(list(string), [])
    service_endpoint_policy_ids = optional(list(string), null)
```